node/api/spend_api_usage.md

```diff
@@ -40,12 +40,12 @@ If you are using the release binaries (i.e. not the Docker image) and you want t
 * For each node, before starting the node edit the user configuration file to specify distinct TCP ports;
 * For each node, before starting the node change the node name of each node.
 
-In order to change the node name of node #1, change directory (`cd`) to the directory where you deployed node #1 then run (change "0.5.0" in the command with the version of the release you deployed):
+In order to change the node name of node #1, change directory (`cd`) to the directory where you deployed node #1 then run (change "2.0.0" in the command with the version of the release you deployed):
```

Comment:
As 2.0.0 has epoch, it may misleading. But this doc will be tagged as (planned) 2.1.0 (current envisaged version number) and note accommodates for version, so ok.